### DISCUSSION TITLE
How to add Login to a sperate deployment.

### DISCUSSION THREAD

**Discussion by Lihfted** [keystonejs/keystone#8980] [Questions]
I have a Backend and a frontend that build separately. Looking for information about adding login and getting the current user on the front end project. I tried looking in documentation and slack but cannot find something relevant. 

This example link is broken https://github.com/keystonejs/keystone/tree/main/examples/with-auth
Ive tried working off : https://github.com/keystonejs/keystone/blob/main/examples/auth/schema.ts

![keystoneAuth](https://github.com/keystonejs/keystone/assets/15110774/f95e7d38-c86c-48a1-bae4-f4fc19b0ffa1)

**Comment by unrevised6419** [comment]
Hey, @Lihfted does this example help you? https://github.com/keystonejs/keystone/tree/main/examples/custom-session-next-auth
